static/app/components/events/contexts/contextIcon.tsx

```diff
@@ -151,9 +150,6 @@ function ContextIcon({name, size: providedSize = 'xl', hideUnknown = false}: Pro
   const extraCass = isDarkmode && INVERT_IN_DARKMODE.includes(name) ? darkCss : null;
 
   const imageName = getLogoImage(name);
-  if (hideUnknown && imageName === logoUnknown) {
-    return null;
-  }
```

Comment:
Removing the `hideUnknown` param because the caller will always receive a <ContextIcon />, it will just render null if specified. If the caller needs to know whether or not the icon will render null, it can just check it manually with `getLogoImage` like I did in `getContextIcon`